test/benchmarks/material/button-harness/protractor-benchmark-utilities.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {runBenchmark} from '@angular/dev-infra-private/benchmark/driver-utilities';
```

Comment:
Are these utilities specific to `button`? Would it make more sense to put this  into something higher, or even in dev-infra? At least as part of this PR it could be moved into `test/benchmarks/utils`?

Reply:
Hmmm, I'm not sure. I created the protractor and testbed benchmark utils so that their tests would be cleaner & simpler.

The testbed one uses `console.warn` which no other test is gonna want to use. The protractor one is more likely to be reusable since we might actually want to get the full time of a test instead of getting the sub-metrics returned by benchpress

I also think I need to go back and try to set up default params for `runBenchmark` again. I tried to do it a while back and surprisingly had a bunch of weird typescript errors throw

Do you think it would be sufficient to try and simplify the `runBenchmark` params instead?

Reply:
Yeah, I think simplifying `runBenchmark` is what we want. I asked in the past for making `ignoreBrowserSynchronization` the default. I'd love us simplifying this.

Reply:
Regardless of the `runBenchmark` API being simpler though; I think it might be good to move these shorthands (for using `console.warn` etc) into a higher-level because I assume we'll have more harness benchmarks, right?

Reply:
Fair enough. I didn't know this is the case. They sounded quite generic.

Reply:
Yeah these are not intended as performance tests for the button harness, they're intended as tests for the harness framework itself, they just happen to be using button to test that. I actually think we should change the directory name to make that more clear. How about `test/benchmarks/cdk/testing/*`?

Reply:
That now makes sense 😄 thanks Miles for clarifying!